engine/gamesys/src/gamesys/scripts/script_tilemap.cpp

```diff
@@ -333,27 +333,7 @@ namespace dmGameSystem
         return 1;
     }
 
-    /*# get a tile from a tile map
-     * Get the tile set at the specified position in the tilemap.
-     * The position is identified by the tile index starting at origin
-     * with index 1, 1. (see [ref:tilemap.set_tile()])
-     * Which tile map and layer to query is identified by the URL and the
-     * layer name parameters.
-     *
-     * @name tilemap.get_tile
-     * @param url [type:string|hash|url] the tile map
-     * @param layer [type:string|hash] name of the layer for the tile
-     * @param x [type:number] x-coordinate of the tile
-     * @param y [type:number] y-coordinate of the tile
-     * @return tile [type:number] index of the tile
-     * @examples
-     *
-     * ```lua
-     * -- get the tile under the player.
-     * local tileno = tilemap.get_tile("/level#tilemap", "foreground", self.player_x, self.player_y)
-     * ```
-     */
-    static int TileMap_GetTile(lua_State* L)
+    static int TileMap_Get(lua_State* L, bool is_full_info)
```

Comment:
These two functions are pretty similar, so it's better to reuse maximum of code here